ddtrace/settings/config.py

```diff
@@ -22,7 +22,12 @@ def __init__(self):
         self._config = {}
         self._http = HttpConfig()
         # Master switch for turning on and off trace search by default
-        self.analytics_enabled = asbool(environ.get('DD_ANALYTICS_ENABLED', False))
+        # accepts both DD_TRACE_ANALYTICS_ENABLED and DD_ANALYTICS_ENABLED
+        # but will prefer the value for DD_TRACE_ANALYTICS_ENABLED every
+        # time
+        self.analytics_enabled = asbool(
+            environ.get('DD_TRACE_ANALYTICS_ENABLED', environ.get('DD_ANALYTICS_ENABLED', False))
```

Comment:
`environ.get()` usage is deprecated internally, sorry :(

We have a `ddtrace.utils.formats.get_env` helper https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/utils/formats.py#L6

It takes two arguments, the first is the "integration name" and the second is the config value, so:

```
get_env('trace', 'analytics_enabled', default=True)
```

Retrieves env variable `DD_TRACE_ANALYTICS_ENABLED`.

For `DD_ANALYTICS_ENABLED`, it looks/feels a little weird, but you can do `get_env('analytics', 'enabled', default=False)`.